debian/control

```diff
@@ -39,15 +52,21 @@ Description: snappy development go packages.
 Package: snapd
 Architecture: any
 Depends: adduser,
+         apparmor (>= 2.10.95-0ubuntu2.2),
          gnupg1 | gnupg,
-         snap-confine (>= 1.0.43),
          squashfs-tools,
          systemd,
          ${misc:Depends},
          ${shlibs:Depends}
-Replaces: ubuntu-snappy (<< 1.9), ubuntu-snappy-cli (<< 1.9)
-Breaks: ubuntu-snappy (<< 1.9), ubuntu-snappy-cli (<< 1.9)
-Conflicts: snap (<< 2013-11-29-1ubuntu1)
+Replaces: snap-confine,
```

Comment:
I think we should build a binary package `snap-confine` out of this source package. The version is higher than the existing snap-confine package so apt will properly upgrade. This is simpler for apt to deal with than the added conflict which may lead to snapd getting held back because apt is conservative and will try to avoid removing packages.